src/modules/colorPicker/ColorPickerUI/App.xaml.cs

```diff
@@ -83,5 +87,10 @@ public void Dispose()
             Dispose(disposing: true);
             GC.SuppressFinalize(this);
         }
+
+        public bool IsRunningAsPowerToy()
```

Comment:
We should come up with a better name, without a context it's kind of cryptic to understand what this means.
`IsRunningDetachedFromPowerToys` may be a better name.

Reply:
Makes sense, but semantically it's a negation of `IsRunningAsPowerToy` so I added the necessary negations.